packages/office-ui-fabric-react/src/components/Button/CommandButton/CommandButton.scss

```diff
@@ -9,61 +9,61 @@
 
 $button-commandButtonHeight: 40px;
 
-.ms-Button--command {
-  @include text-align(left);
+.ms-Button {
+  &.ms-Button--command {
```

Comment:
basically if something is defined in `ms-Button`, and overridden in `ms-Button--command`, we will run into race conditions.